pepys_import/file/e_trac_importer.py

```diff
@@ -36,12 +34,7 @@ def can_load_this_file(self, file_contents):
         return True
 
     def load_this_file(self, data_store, path, file_object, datafile):
-        self.errors = list()
-        basename = os.path.basename(path)
-        print(f"E-trac parser working on {basename}")
-        error_type = self.short_name + f" - Parsing error on {basename}"
-        prev_location = dict()
-        datafile.measurements[self.short_name] = list()
+        super().load_this_file(data_store, path, file_object, datafile)
```

Comment:
One alternate strategy here.

An issue is that the parser writer may _forget_ to call this super-method.

What if the parent class has a method, which then calls this method?

So, in `file_processor` we call something like `core_load_this_file()` - which performs those init steps, and then it calls `load_this_file()`.  That avoids the requirement for the parser writer to call the parent.

Yes - that naming could be improved ;-)

Reply:
Good idea. Done. Just waiting for tests to pass and then it'll be ready to merge.